### PR TITLE
Update API documentation with updated limits of size and # for file attachments

### PR DESCRIPTION
**Pull request by jimleroyer**
# Summary | Résumé

Added warning on current temp file size and max file number, to better inform our users with latest changes in Notify and temporary ones.

[Related zenhub card](https://app.zenhub.com/workspaces/notify-planning-614b3ad91bc2030015ed22f5/issues/cds-snc/notification-planning/659).

# Test instructions | Instructions pour tester la modification

Head over to the review app link. Specifically to these sections:

* [Sending a file by email](https://cds-snc.github.io/notification-documentation/add-file-num-max-and-current-restriction/review-apps/7e49dc/en/send.html#sending-a-file-by-email)
* [Envoyer un fichier par courriel](https://cds-snc.github.io/notification-documentation/add-file-num-max-and-current-restriction/review-apps/7e49dc/fr/envoyer.html#envoyer-un-fichier-par-courriel)

### CODE REVIEW

**Comment by anikbrazeau** [comment]
> Due to a recent infrastructure change~s~, the current maximum size~s~ for all files and content within an email notification must be smaller than 6MB.

@yaelberger-commits, FR to match:
`En raison d'un récent changement d'infrastructure, la taille maximale actuelle de tous les fichiers et du contenu du courriel doit être inférieure à 6 Mo.`

**Comment by YedidaZalik** [comment]
Question: we've changed the non-API terminology from "personalization" to "custom content". Would it be worthwhile to change that here as well to align vocab and facilitate communication between non-dev and dev team members sending messages?

**Comment by jimleroyer** [comment]
I fixed the typos @yaelberger-commits @anikbrazeau

**Comment by jimleroyer** [comment]
@yaelberger-commits @YedidaZalik Regarding the _personalisation_ vocabulary, I'd process it in a separate card. We might want to discuss what we will change and what we will keep. The _personalisation_ word is used throughout the API requests and we won't be able to easily change this in the actual API calls, but we could change a few places within the documentation to better fit the rest of the app.

**Comment by jimleroyer** [comment]
@YedidaZalik I changed the code owners so that you receive the content update notifications on this repository in the future, when documentation will get updated.

**Comment by YedidaZalik** [comment]
@jimleroyer @yaelberger-commits Thanks and I'm sorry I did not get to this yesterday. 
I've read these notes and the page now, but this is my first time as a reviewer on the API. To make sure that I'm doing what I'm supposed to do, may I sync with one of you (I know you're both very busy right now)?

**Comment by jimleroyer** [comment]
No problem @YedidaZalik ! Meeting with us would expedite this. Feel free to schedule a 15-30 minutes session with me or come to gathertown when we got the emoji in our Slack status. 👍

**Comment by YedidaZalik** [comment]
Thanks @jimleroyer 
Given today's power outage, I've scheduled for Monday. Please feel free to change date and time so it works for you!

**Comment by YedidaZalik** [comment]
@jimleroyer -  some typos in :
1- Temporary restriction on overall files size

Due to a recent infrastructure changes, the current maximum sizes for all files and content within an email notification must be smaller than 6MB.

Please change to:

Temporary restriction on overall file size

Due to recent infrastructure changes, the current maximum size for all files and content within an email notification must be smaller than 6MB.


2 - Please change Error codes
If the request is not successful, the response body is json, refer to the table below for details.
TO:
Error codes
If the request is not successful, the response body is json. Refer to the table below for details.

@jimleroyer did I do this correctly?

**Comment by jimleroyer** [comment]
@YedidaZalik All good 👍 , just pushed changes with your corrections.

**Comment by adriannelee** [comment]
I'm supposed to be seeing the deleted text as crossed out text, right? Other wise, lgtm!

File: src/fr/envoyer.md [hunk 1]
```diff
@@ -93,10 +93,16 @@ Cette fonctionnalité n’est disponible que par le biais de l’API. Pour activ
...
 ### Types de fichiers et prérequis de taille
-Vous pouvez téléverser des fichiers PDF, CSV, .jpeg, .png, .odt, .txt, .rtf, et des fichiers Microsoft Excel et Microsoft Word. La taille de votre fichier et de votre courriel doit être inférieure à 10 Mo.
+
+Vous pouvez téléverser des fichiers PDF, CSV, .jpeg, .png, .odt, .txt, .rtf, et des fichiers Microsoft Excel et Microsoft Word. Un maximum de 10 pièces jointes est autorisé par courriel. ~~La taille de votre fichier et de votre courriel doit être inférieure à 10 Mo~~.
 
 Si vous devez envoyer d’autres types de fichiers, [communiquez avec nous](https://notification.canada.ca/contact?lang=fr) .
 
+::: warning Restriction temporaire sur la taille des fichiers
+
+En raison d'un récent changement d'infrastructure, la taille maximale actuelle de tous les fichiers et du contenu du courriel doit être inférieure à 6 Mo.
```
[the remark this answers is not in the document]
Comment (jimleroyer) [reply src/fr/envoyer.md:103]: * On a changé l'infrastructure, j'ai changé à "En raison d'un récent changement dans notre infrastructure"
* Idéalement on répare bientôt et c'est temporaire.
* "d'un courriel": Changé. 👍

[the remark this answers is not in the document]
Comment (jimleroyer) [reply src/fr/envoyer.md:103]: Je vais ouvrir un autre PR parce que ça été mergé:
https://github.com/cds-snc/notification-documentation/pull/92

**Pull request merged**